android/src/main/java/com/novoda/notils/orientation/ScreenOrientationLockerFactory.java

```diff
@@ -0,0 +1,44 @@
+package com.novoda.notils.orientation;
+
+import android.content.res.Resources;
+
+import com.novoda.android.BuildConfig;
+import com.novoda.android.R;
+
+/**
+ * This factory can be used to instantiate a {@link ScreenOrientationLocker}
+ * alternatively the {@link OrientationLockedActivity} can be extended for the same behaviour
+ */
+public class ScreenOrientationLockerFactory {
+
+    private final boolean isDebug;
+    private final boolean isTablet;
+
+    public static ScreenOrientationLockerFactory newInstance(Resources resources) {
+        boolean isDebug = BuildConfig.DEBUG;
```

Comment:
For libraries, this is always `false` by default. You can enable debug and release builds for libraries but it does not make sense when you publish them.